packages/fxa-shared/metrics/glean/web/index.ts

```diff
@@ -50,6 +58,7 @@ export const eventsMap = {
     view: 'cached_login_view',
     submit: 'cached_login_submit',
     success: 'cached_login_success_view',
+    forgotPassword: 'cached_login_forgot_pwd_submit',
```

Comment:
Caught this because of the change on https://github.com/mozilla/fxa/pull/16668/files#diff-dc1ed2b3d52dd10d854ded9e3d925ef9c9b7e229ff4e83560e1d944b6110a03bR51